kernel/packages/entryPoints/website.ts

```diff
@@ -141,23 +142,29 @@ namespace webApp {
         i.ConfigureHUDElement(HUDElementID.USERS_AROUND_LIST_HUD, { active: voiceChatEnabled, visible: false })
         i.ConfigureHUDElement(HUDElementID.FRIENDS, { active: identity.hasConnectedWeb3, visible: false })
 
-        ensureRendererEnabled().then(() => {
-          globalThis.globalStore.dispatch(setLoadingWaitTutorial(false))
-          globalThis.globalStore.dispatch(experienceStarted())
-          globalThis.globalStore.dispatch(setLoadingScreen(false))
-          Html.switchGameContainer(true)
-        }).catch(logger.error)
+        ensureRendererEnabled()
+          .then(() => {
+            globalThis.globalStore.dispatch(setLoadingWaitTutorial(false))
+            globalThis.globalStore.dispatch(experienceStarted())
+            globalThis.globalStore.dispatch(setLoadingScreen(false))
+            Html.switchGameContainer(true)
+          })
+          .catch(logger.error)
 
         const tutorialConfig = {
           fromDeepLink: HAS_INITIAL_POSITION_MARK,
-          enableNewTutorialCamera: enableNewTutorialCamera,
+          enableNewTutorialCamera: enableNewTutorialCamera
         }
 
         EnsureProfile(identity.address)
           .then((profile) => {
             i.ConfigureEmailPrompt(profile.tutorialStep)
             i.ConfigureTutorial(profile.tutorialStep, tutorialConfig)
             i.ConfigureHUDElement(HUDElementID.GRAPHIC_CARD_WARNING, { active: true, visible: true })
+
+            if (isWaitingTutorial(globalThis.globalStore.getState())) {
+              teleportObservable.notifyObservers(worldToGrid(lastPlayerPosition))
```

Comment:
Is this how the renderer is activated?

Reply:
Yes, if scene is already loaded it will activate the renderer there, if scene is not loaded then it will wait until loading finishes

Reply:
Could we move this (and the same call below) to a new function named `activateRenderer` or something like that? Because I'm sure I'll forget what this does in a few months